app/assets/javascripts/spree/frontend/spree_quotes_management.js

```diff
@@ -1,2 +1,35 @@
 // Placeholder manifest file.
-// the installer will append this file to the app vendored assets here: vendor/assets/javascripts/spree/frontend/all.js'
\ No newline at end of file
+// the installer will append this file to the app vendored assets here: vendor/assets/javascripts/spree/frontend/all.js'
+
+
+function SpreeQuotesManagement(max_limit, span_char_left, quotes_textarea) {
```

Comment:
receive id of the div here and find all the elements related to it in constructor.
Also set default value of max_limit through the constant defined in the model.